backend/main.py

```diff
@@ -23,6 +23,9 @@ async def homepage(request):
 async def school(request):
     return JSONResponse({'lesson': 'software engineering'})
 
+async def shanties(request):
+    f = open('./shanties.txt', 'r')
```

Comment:
It is a good practice to always close the file when you are done with it. Read about context manager with open or other solution. Pls do some time tests for diffrent solutions.